test.e2e/spec/services.spec.js

```diff
@@ -471,6 +471,49 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
       });
     });
 
+    describe('$resource remoteMethod with array descriptions', function() {
+      var $injector;
+      before(function() {
+        return given.servicesForLoopBackApp(
+          {
+            models: {
+              'remotable': {},
+            },
+            setupFn: (function(app, cb) {
+              var Remotable = app.models.Remotable;
+              Remotable.arrayDescription = function(cb) {
+                cb(null, {});
+              };
+              Remotable.remoteMethod(
+                'arrayDescription', {
+                  http: { path: '/arrayDescription', verb: 'POST' },
+                  accpets: [{
```

Comment:
>Please make sure you see the test fail first.>
i did test with the previous implementation, and return was causing it to fail apparently and therefore didnt notice the `accepts` was misspelled 

>IIRC, LoopBack built-in model methods have multiple accepts arguments (i.e. an array), so this case should be already covered by existing tests.
right, will fix to only use one argument

the remoteMethod description itself uses the [`ejs filter`](https://github.com/strongloop/loopback-sdk-angular/blob/57b7a486af1e72736b117764456f32116e812fd5/lib/services.template.ejs#L531) which automatically handles strings, 
while the [postData (aka: accepts)](https://github.com/strongloop/loopback-sdk-angular/blob/57b7a486af1e72736b117764456f32116e812fd5/lib/services.template.ejs#L554)[2](https://github.com/strongloop/loopback-sdk-angular/blob/57b7a486af1e72736b117764456f32116e812fd5/lib/services.template.ejs#L579) and [returns](https://github.com/strongloop/loopback-sdk-angular/blob/57b7a486af1e72736b117764456f32116e812fd5/lib/services.template.ejs#L612) does a javascript syntax which fails to `[].replace()` 

prior to f56b103da4ca7ab0e311402a814d18e49db192ce it was using `(arg.description || '').replace(/\n/g, '\n         *   ')` [which fails](http://ci.strongloop.com/job/loopback-sdk-angular/node=0.12,os=windows/2700/console)

```
annot generate services script: TypeError: ejs:489
    487| <% if (action.deprecated) { -%>
    488|          * @deprecated <%- action.deprecated %>
 >> 489| <% } -%>
    490|          *
    491|          * @description
    492|          *

(arg.description || "").replace is not a function
    at eval (eval at <anonymous> (/Users/davidcheung/node-web/loopback-sdk-angular/node_modules/ejs/lib/ejs.js:242:14), <anonymous>:105:25)
    at Array.forEach (native)
    at ngdocForMethod (eval at <anonymous> (/Users/davidcheung/node-web/loopback-sdk-angular/node_modules/ejs/lib/ejs.js:242:14), <anonymous>:104:10)
    at eval (eval at <anonymous> (/Users/davidcheung/node-web/loopback-sdk-angular/node_modules/ejs/lib/ejs.js:242:14), <anonymous>:41:92)
    at Array.forEach (native)
    at eval (eval at <anonymous> (/Users/davidcheung/node-web/loopback-sdk-angular/node_modules/ejs/lib/ejs.js:242:14), <anonymous>:40:87)
    at eval (eval at <anonymous> (/Users/davidcheung/node-web/loopback-sdk-angular/node_modules/ejs/lib/ejs.js:242:14), <anonymous>:111:48)
    at /Users/davidcheung/node-web/loopback-sdk-angular/node_modules/ejs/lib/ejs.js:255:15
    at Object.exports.render (/Users/davidcheung/node-web/loopback-sdk-angular/node_modules/ejs/lib/ejs.js:293:13)
    at Object.generateServices [as services] (/Users/davidcheung/node-web/loopback-sdk-angular/lib/services.js:93:14)
POST /setup 200 17.411 ms - 142
GET /services?services::$resource%20for%20model%20with%20funky%20name::has%20a%20factory%20name%20that%20starts%20with%20upper-case 200 3.521 ms - 196
    $resource remoteMethod with array descriptions
```